notifications_utils/recipients.py

```diff
@@ -44,10 +44,10 @@
 }
 
 # regexes for use in validate_email_address.
-# invalid local chars - whitespace, quotes and apostrophes, semicolons and colons, GBP sign
+# Valid characters taken from https://en.wikipedia.org/wiki/Email_address#Local-part
 # Note: Normal apostrophe eg `Firstname-o'surname@domain.com` is allowed.
-INVALID_LOCAL_CHARS = r"\s\",;:@£“”‘’"
-email_regex = re.compile(r'^[^{}]+@([^.@][^@]+)$'.format(INVALID_LOCAL_CHARS))
+VALID_LOCAL_CHARS = r"a-zA-Z0-9.!#$%&'*+/=?^_`{|}~\-"
```

Comment:
TIL that all these symbols are valid in the address 😱 🤔 

Edit:
However, reading the link above it says that some of them are only valid if quoted. Probably not something to be implemented now, but useful to keep in mind for a future iteration as and when necessary.

Reply:
Shouldn't be! Which ones did you see? 

wikipedia lists this one `#!$%&'*+-/=?^_`{}|~@example.org` as a completely valid unquoted address.

Reply:
Ah you're right; I read the bit below, but didn't notice the different character subset:

> space and "(),:;<>@[\] characters are allowed with restrictions (they are only allowed inside a quoted string, as described in the paragraph below, and in addition, a backslash or double-quote must be preceded by a backslash);

However it says that `..` is allowed if quoted:

> dot ., provided that it is not the first or last character unless quoted, and provided also that it does not appear consecutively unless quoted (e.g. John..Doe@example.com is not allowed but "John..Doe"@example.com is allowed);[6]

Reply:
yeah i decided not to allow quoted local parts because i don't think i've ever seen an actual email address